app/poros/user_event.rb

```diff
@@ -0,0 +1,15 @@
+class UserEvent
+  attr_reader :id, :user_id, :gym_id, :date_time, :activity
+
+  def initialize(info)
+    @id = info[:id]
+    @user_id = info[:attributes][:user_id]
+    @gym_id = info[:attributes][:gym_id]
```

Comment:
With the new schema, the `gym_id` is now `gym_membership_id`

Reply:
[Issue](https://github.com/tvaroglu/spot_me_frontend/issues/20) opened